src/System.Json/src/System.Json.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <AssemblyVersion>4.0.0.0</AssemblyVersion>
+    <ProjectGuid>{2ADDB484-6F57-4D71-A3FE-A57EC6329A2B}</ProjectGuid>
+    <PackageTargetFramework>netstandard1.3</PackageTargetFramework>
```

Comment:
I was having trouble getting all of the various things to match up, e.g. the target framework, the supported frameworks entry, etc.  I expect it'd be doable, I just didn't know what to put in all of the various places and ended up copying this from elsewhere.  If I make this 1.0, what should I put in those other places, e.g. SupportedFrameworks?